packages/backend/src/peripherals/database/PreprocessedAssetHistoryRepository.ts

```diff
@@ -6,7 +6,9 @@ import { Logger } from '../../tools/Logger'
 import { BaseRepository } from './shared/BaseRepository'
 import { Database } from './shared/Database'
 
-export interface PreprocessedAssetHistoryRecord<T extends AssetHash | AssetId> {
+export interface PreprocessedAssetHistoryRecord<
+  T extends AssetHash | AssetId = AssetHash | AssetId
```

Comment:
I don't understand what is happening here :) AssetId in perpetuals (e.g. "ETH-9") is different then AssetHash on spot (e.g. "0xe0ab3eef.."). Losing this distinction here can prevent TypeScript from properly ensuring types in other places. But I'm not sure if this is the case here.